lib/linguist/languages.yml

```diff
@@ -4343,6 +4343,17 @@ STON:
   tm_scope: source.smalltalk
   ace_mode: text
   language_id: 336
+Svelte:
```

Comment:
Items in `languages.yml` should be alphabetically sorted, with uppercase letters first.